common_features.mk

```diff
@@ -619,6 +619,11 @@ ifeq ($(strip $(AUTO_SHIFT_ENABLE)), yes)
     endif
 endif
 
+ifeq ($(strip $(TAP_TERM_KEYS_ENABLE)), yes)
+    SRC += $(QUANTUM_DIR)/process_keycode/process_tap_term_keys.c
+    OPT_DEFS += -DTAP_TERM_KEYS_ENABLE
+endif
+
```

Comment:
This section can now be replaced by adding `TAP_TERM_KEYS` to the list in `builddefs/generic_features.mk` -- it'll automatically do what you've done here.

Reply:
Solved in https://github.com/qmk/qmk_firmware/pull/11036/commits/725ea90ee0d5782149a5caec7c1f15826f8d0b13